pkg/binlog/reader/tcp_test.go

```diff
@@ -90,10 +90,23 @@ func (t *testTCPReaderSuite) setUpData(c *C) {
 	query := fmt.Sprintf("DROP DATABASE `%s`", dbName)
 	_, err := t.db.Exec(query)
 
-	// delete previous binlog files/events.
-	query = "RESET MASTER"
-	_, err = t.db.Exec(query)
-	c.Assert(err, IsNil)
+	maxRetryCount := 5
+	for i := 0; i < maxRetryCount; i++ {
```

Comment:
maybe we can have a function in `pkg/unit`, like 
```
func WaitSomething(backoff int, fn func() bool) bool {
	for i := 0; i < backoff; i++ {
		if fn() {
			return true
		}

		time.Sleep(10 * time.Millisecond)
	}

	return false
}
```

Reply:
done in https://github.com/pingcap/dm/pull/171/commits/6c3fab5c440efc069bd43a66c762a980909b9c2a.